.travis.yml

```diff
@@ -19,6 +17,12 @@ env:
 
 matrix:
   include:
+    - php: 5.4
+        dist: trusty
```

Comment:
this indentation is probably the error

Reply:
You're right. Thank you.

Fixed now. Will see what actual build will show.